SYCL/Sampler/normalized-mirror-linear-float.cpp

```diff
@@ -18,21 +17,20 @@
 
 */
 
+#include "common.hpp"
 #include <CL/sycl.hpp>
 
 using namespace cl::sycl;
 
 // pixel data-type for RGBA operations (which is the minimum image type)
 using pixelT = sycl::float4;
 
-// will output a pixel as {r,g,b,a}.  provide override if a different pixelT is
-// defined.
-void outputPixel(sycl::float4 somePixel) {
-  std::cout << "{" << somePixel[0] << "," << somePixel[1] << "," << somePixel[2]
-            << "," << somePixel[3] << "} ";
-}
-
 // some constants.
+pixelT ref[] = {
```

Comment:
So functionally, this is fine. But the problem is that we are losing all the context that is relevant to debugging. For example in this array, the first three pixels were read at the inner boundary locations, the next four above the right bound, the next four below the left bound.   I would like it if these tests brought the comments that we used to see in the output into here.  It is important because not all these samplers are working on all platforms devices and backends, so understanding what type of read operation is failing is crucial to fixing and verifying them.

Reply:
I split `ref` into several vectors with comments according to their content. Please check if it is ok